src/components/Post/Comments/Comments.js

```diff
@@ -1,25 +1,48 @@
 import React from 'react';
 import { graphql, StaticQuery } from 'gatsby';
 import ReactDisqusComments from 'react-disqus-comments';
+import md5 from 'md5';
+import 'gitalk/dist/gitalk.css';
```

Comment:
Styles need to be moved to _base.scss ([src/assets/scss/_base.scss](https://github.com/alxshelepenok/gatsby-starter-lumen/blob/master/src/assets/scss/_base.scss))